src/App.tsx

```diff
@@ -10,33 +12,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTab] = React.useState(tabs[0].id);
+
+  const onTabSelected = (event: Tab) => {
+    setSelectedTab(event.id);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${(selectedTabId
+          && (tabs.find((tab: Tab) => tab.id === selectedTabId))?.title)}`}
       </h1>
```

Comment:
if you saved the selectedTab in the App, you could just write here Selected tab is selectedTab.content. 
p.s. It was written in the Readme file. (Save the `selectedTab` in the `App` (the first `tab` is selected by default);)